locations/tempelwg.yml

```diff
@@ -0,0 +1,159 @@
+---
+location: tempelwg
+location_nice: U-Alt Tempelhof
+latitude: 52.465551
+longitude: 13.38598
+contacts:
+  - "@nick:matrix.riotcat.org"
+
+hosts:
+  - hostname: tempelwg-core
+    role: corerouter
+    model: "dlink_covr-x1860-a1"
+    wireless_profile: tempelwg
+
+  - hostname: tempelwg-ap-tini
+    role: ap
+    model: "totolink_a7000r"
+    wireless_profile: tempelwg
+
+ipv6_prefix: "2001:bf7:810:1500::/56"
+
+dhcp_no_ping: false
+
+# Reserved Prefixes:
+# 10.248.17.0/26, 2001:bf7:810:1500::/56
+# MGMT: 10.248.17.0/28
+# MESH: 10.248.17.16/26
+# DHCP: 10.248.17.32/27
+
+networks:
+
+  # MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_5g
+    prefix: 10.248.17.16/32
+    ipv6_subprefix: -20
+    mesh_ap: tempelwg-core
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_2g
+    prefix: 10.248.17.17/32
+    ipv6_subprefix: -21
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: tempelwg-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  - vid: 42
+    role: mgmt
+    prefix: 10.248.17.0/28
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      tempelwg-core: 1
+      tempelwg-ap-tini: 2
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.248.17.32/27
+    ipv6_subprefix: 0
+    inbound_filtering: true
+    enforce_client_isolation: true
+    assignments:
+      tempelwg-core: 1
+
+  - vid: 50
+    name: prdhcp
+    role: uplink
+    untagged: true
+
+  - role: tunnel
+    ifname: ts_wg0
+    mtu: 1280
+    prefix: 10.248.17.18/32
+    wireguard_port: 51820
+
+  - role: tunnel
+    ifname: ts_wg1
+    mtu: 1280
+    prefix: 10.248.17.19/32
+    wireguard_port: 51821
+
+location__channel_assignments_11g_standard__to_merge:
+  tempelwg-core: 13-20
+  tempelwg-ap-tini: 1-20
+
+location__channel_assignments_11a_standard__to_merge:
+  tempelwg-core: 36-20
```

Comment:
Why only 20 MHz channel width? Intentional even with access to a private network?